bin/mail/2016-05-02-petition-followup/generate-list.rb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+
+# All US based teachers and HOC organizers who have not clicked through the previous petition email.
+
+# First, get all US teachers and past organizers.
+teacher_query = %q(
+  ((kind_s:user && role_s:teacher) ||
+  kind_s:HocSignup2015 || kind_s:HocSignup2014 || kind_s:"CSEdWeekEvent2013") && (
+    create_ip_country_s: "United States" ||
+    hoc_country_s: "us" ||
+    location_country_code_s: "US"
+  )
+)
+
+fields = %w[email_s name_s]
+teachers = query_subscribed_contacts(q: teacher_query, fl: fields)
+puts "#{teachers.length} total US teachers and past HOC organizers."
+
+
+# Next, filter for those who did not click through the previous email.
+PREVIOUS_MESSAGE_NAME = '4-22-petition-congress-3b-no'
```

Comment:
No, because this only applies to the final email template '4-22-petition-congress-3b-no'. But you raise a good point. We will not be re-sending to any of the experimental emails. I'll follow up with marketing to see if we should do those too.